mangaki/mangaki/wrappers/anilist.py

```diff
@@ -513,25 +513,20 @@ def get_user_list(self,
 )
 
 anilist_langs = AniListLanguages()
-language_map = {
-    'english': anilist_langs.english_ext_lang,
-    'romaji': anilist_langs.romaji_ext_lang,
-    'japanese': anilist_langs.japanese_ext_lang,
-    'unknown': anilist_langs.unknown_ext_lang,
-}
 
 work_categories = WorkCategories()
-category_map = {
-    AniListWorks.animes: work_categories.anime,
-    AniListWorks.mangas: work_categories.manga
-}
 
 staff_roles = StaffRoles()
-role_map = staff_roles.role_map
 
 
 def build_work_titles(work: Work,
                       titles: Dict[str, Tuple[str, str]]) -> List[WorkTitle]:
+    language_map = {
```

Comment:
No, please, there is a reason for them to be outside. They might be reused outside, and so on.

Reply:
I had errors because of that. Because they require items that are not initialized in the tests. Are you sure they should be outside?

At some point I had errors like "database does not exist".

Reply:
I checked. It is not used anywhere. Is it okay?

Reply:
We'll see later. AniList is not used yet, but should be soon.